package/MDAnalysis/core/groups.py

```diff
@@ -1487,12 +1471,46 @@ class AtomGroup(GroupBase):
     AtomGroup instances are always bound to a
     :class:`MDAnalysis.core.universe.Universe`. They cannot exist in isolation.
 
+
+    .. rubric:: Deprecated functionality
+
+    *Instant selectors* will be removed in the 1.0 release.  See issue `#1377
+    <https://github.com/MDAnalysis/mdanalysis/issues/1377>`_ for more details.
+
+    Atoms can also be accessed in a Pythonic fashion by using the atom name as
+    an attribute. For instance, ::
+
+        ag.CA
+
+    will provide a :class:`AtomGroup` of all CA atoms in the
+    group. These *instant selector* attributes are auto-generated for
+    each atom name encountered in the group.
+
+    .. note::
+
+        The name-attribute instant selector access to atoms is mainly
+        meant for quick interactive work. Thus it either returns a
+        single :class:`Atom` if there is only one matching atom, *or* a
+        new :class:`AtomGroup` for multiple matches.  This makes it
+        difficult to use the feature consistently in scripts.
+
+
     See Also
     --------
     :class:`MDAnalysis.core.universe.Universe`
 
+
+    .. deprecated:: 0.16.2
+       *Instant selectors* of AtomGroup will be removed in the 1.0 release.
+       See issue `#1377
+       <https://github.com/MDAnalysis/mdanalysis/issues/1377>`_ for
+       more details.
+
     """
     def __getitem__(self, item):
+        # DEPRECATED in 0.16.2
+        # REMOVE in 1.0
+        #
```

Comment:
This will annoy the hell of people, won't it? I can't use the decorator because `__getitem__` is used for other stuff.